### ISSUE TITLE
How to judge the "significantly different in IID performance with our baseline"?

### ISSUE THREAD

**Issue by FrankZhangRp** [eccv22-ood-workshop/ROBIN-dataset#13]
Hi,
Is the IID performance of the released dataset the same as the paper?
![image](https://user-images.githubusercontent.com/32047074/180724937-6abc5912-36e2-4857-a71e-0a762736e2c3.png)
There are two different backbone networks in the paper, are the backbone networks fixed in this challenge? And how to judge the "significantly different in IID performance with our baseline"? 
Thanks a lot!

**Comment by wjm-wjm** [comment]
Hi, I find that the Top-1 performance of image classification approach to 100% after training on both iid_test and nuisances validation sets. Have you got this problem ?

**Comment by FrankZhangRp** [comment]
I think the training is only on the train/Images and both iid_test and nuisances are just used for testing. And I am also confused of the model selection strategy, I think the model can only be selected by the iid_test set to evaluate the algorithms' OOD performance .
Here are the baseline results of our implementation.
<img width="474" alt="image" src="https://user-images.githubusercontent.com/32047074/180904569-74a473f8-a510-4784-a8b8-e79245e50869.png">

**Comment by wjm-wjm** [comment]
I have run following command: "python main.py --data /home/wjm/wjm/dataset/ROBINv1.0/train --val-data /home
/wjm/wjm/dataset/ROBINv1.0/nuisances --pretrained"
![EFQUZ2WR%J~ZSQD@YGWWG8L](https://user-images.githubusercontent.com/53390698/180908619-2aa94b36-b10e-4586-94d7-bbab3014faa3.png)
The performance seems to approach 100% after one epoch training, is something wrong happening？Thanks for your kind help!

**Comment by DTennant** [comment]
> I think the training is only on the train/Images and both iid_test and nuisances are just used for testing. And I am also confused of the model selection strategy, I think the model can only be selected by the iid_test set to evaluate the algorithms' OOD performance . Here are the baseline results of our implementation. <img alt="image" width="474" src="https://user-images.githubusercontent.com/32047074/180904569-74a473f8-a510-4784-a8b8-e79245e50869.png">

Hi, The detail of the model selection is still not confirmed by our sponsor, but we are planning to implement it as this:
we have an updated mean and variance of the top-1 accuracy on our dataset, and for each team participating in our challenge, the final performance should be within the range of mean +- 3 * variance, and our baseline will be the ResNet50 results in the paper.




> I have run following command: "python main.py --data /home/wjm/wjm/dataset/ROBINv1.0/train --val-data /home /wjm/wjm/dataset/ROBINv1.0/nuisances --pretrained" ![EFQUZ2WR%J~ZSQD@YGWWG8L](https://user-images.githubusercontent.com/53390698/180908619-2aa94b36-b10e-4586-94d7-bbab3014faa3.png) The performance seems to approach 100% after one epoch training, is something wrong happening？Thanks for your kind help!

Hi, the data processing script and the baselines for the three tasks will be released after we have confirmation from our sponsor, thanks for you interest.

**Comment by DTennant** [comment]
Hi,
I have updated the dataset and uploaded the evaluation code that we will be using for the codalab server, you can check the code for your questions.
